specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2022-10-01-preview/examples/GetClusterExample.json

```diff
@@ -32,7 +32,20 @@
           "totalNodeCount": 2,
           "agentVersion": "0.1.0",
           "privateLinkState": "Disabled",
-          "provisioningState": "Succeeded"
+          "provisioningState": "Succeeded",
+          "azureHybridBenefit": "NotApplicable",
+          "distribution": "AKS", 
+          "distributionVersion": "1.0",
+          "additionalProperties": [
```

Comment:
Isn't additionalProperties a reserved keyword? why are we using it as a property name in the examples?

Reply:
Changed to miscellaneousProperties.